firecloud_project_test.py

```diff
@@ -184,6 +204,37 @@ def test_pubsub_notifications(self):
     self.assertEqual(completed['metadata']['dependsOn'],
                      '$(ref.fc-network.resourceNames)')
 
+  def test_satisfy_label_requiements(self):
```

Comment:
spelling: missing an 'r' in requirements here and in the function in `firecloud_project.py`

Reply:
🦅  👁